src/widgets/button.cpp

```diff
@@ -166,7 +166,10 @@ namespace gcn
         graphics->drawLine(getWidth() - 1, 1, getWidth() - 1, getHeight() - 1);
         graphics->drawLine(1, getHeight() - 1, getWidth() - 1, getHeight() - 1);
 
+		if (isEnabled())
```

Comment:
spacing is weird

Reply:
Fixed with 25c670187742a8c0dbe560b120b257c5bc93993f